pages/Posts.tsx

```diff
@@ -0,0 +1,114 @@
+import { useState, useEffect, useRef } from "react";
+import { inter, roboto_serif } from "../styles/fonts";
+
+//Api
+const POSTS_URL = "https://jsonplaceholder.typicode.com/posts?_page=";
+
+
+// Type definitions
+type Post = {
+  id: number;
+  title: string;
+  body: string;
+  userId: number;
+};
+
+type Error = {
+  message: string;
+};
+type PageNumber = number;
```

Comment:
You did not need to create a type alias for this.

Typing line 26 as `number` would have been enough